assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -84,9 +125,14 @@ const WPDashboardWidgets = () => {
 					) }
 				</div>
 			) }
-			{ analyticsModuleActive && analyticsModuleConnected && (
-				<WPDashboardPopularPagesWidget />
-			) }
+
+			{ analyticsModuleActive && analyticsModuleConnected && ! shouldCombineAnalyticsArea2 && <WPDashboardPopularPagesWidget /> }
+
+			<div className={ HIDDEN_CLASS }>
```

Comment:
Nit-pick, but maybe we could only render that if it's needed (i.e. if at least one of the below conditions is true).